README.md

```diff
@@ -81,6 +81,15 @@ $details->country_name; // United States
 
 ```php
 $details->is_eu; // False
+```
+#### Country Flag
+
+``Details->country_flag`` will return emoji and unicode of country's flag, as supplied by the ``flags.json`` file.
```

Comment:
```suggestion
```

#### Country Flag

`details->country_flag` will return the emoji and unicode representations of the country's flag, as supplied by the `flags.json` file.
```

Reply:
Ok that didn't work.